debs/SPECS/4.2.0/wazuh-agent/debian/rules

```diff
@@ -60,10 +60,14 @@ override_dh_install:
 
 	# Copying init.d script
 	mkdir -p ${TARGET_DIR}/etc/init.d/
-	cp src/init/ossec-hids-debian.init ${TARGET_DIR}/etc/init.d/wazuh-agent
+	sed "s/WAZUH_HOME_TMP/${INSTALLATION_DIR}/g" src/init/templates/ossec-hids-debian.init > src/init/templates/ossec-hids-debian.init.tmp
+	mv src/init/templates/ossec-hids-debian.init.tmp src/init/templates/ossec-hids-debian.init
```

Comment:
For deb and rpm you can use `sed -i expression` since this implementation works fine with the `-i` parameter. However, you may need to implement the change requested before in the AIX spec.

Reply:
Done, ty, https://github.com/wazuh/wazuh-packages/pull/609/commits/09b244febb60d640fd92f0af007f7120e3ec2332